src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -571,6 +615,16 @@ class AddUserProfile extends Component {
                     },
                   });
                   break;
+                case 'name':
```

Comment:
Can you include break statement? When the if condition fails, the switch doesn't stop.